tests/tests_common_test.go

```diff
@@ -394,3 +396,35 @@ func addDomainResourcesToVMI(vmi *kubevirtv1.VirtualMachineInstance, cpuCores ui
 	}
 	return vmi
 }
+
```

Comment:
All these functions are only used in one place, can you inline them?

Reply:
Done